packages/x-components/src/x-modules/queries-preview/store/types.ts

```diff
@@ -0,0 +1,53 @@
+import { XActionContext, XStoreModule } from '../../../store/index';
+
+/**
+ * QueriesPreview store state.
+ *
+ * @public
+ */
+export interface QueriesPreviewState {}
+
+/**
+ * QueriesPreview store actions.
+ *
+ * @public
+ */
+export interface QueriesPreviewActions {}
```

Comment:
Can you move the actions under the mutations? 🙏 The other modules use that order.

Reply:
Sure!